### ISSUE TITLE
Update youtube-dl version

### ISSUE THREAD

**Issue by jcohenho** [layer8x/youtube-dl.rb#55]
What version of youtube-dl is this gem using? Can this be upgraded to the latest version?

**Comment by mihairadulescu** [comment]
yeah dude there is a pr with updated sources... can you update this?
